src/SequenceManager.cc

```diff
@@ -170,56 +169,14 @@ std::vector<SequenceManager::SequenceOnDisk> SequenceManager::list_sequences() c
 
         auto rel_path = std::filesystem::relative(entry.path(), path_);
 
+        // Accept only folders with a valid name and unique ID
         if (seq_info.name.has_value() && seq_info.unique_id.has_value())
         {
             sequences.push_back(SequenceOnDisk{
                 std::move(rel_path),
                 std::move(*seq_info.name),
```

Comment:
Maybe does not belong here, but (some weak comment)
* first I thought _why is this `*.` not `->`
* found out it is an optional
* maybe use instead:

```suggestion
                std::move(seq_info.name.value()),
```

Just to make it easier to comprehend.